engine/src/main/java/org/hibernate/search/util/impl/ReflectionHelper.java

```diff
@@ -115,13 +115,34 @@ public static Object getMemberValue(Object bean, XMember getter) {
 	}
 
 	/**
-	 * Checks whether the specified class contains any Search specific annotations.
+	 * Checks whether the specified class contains any Search-specific annotations, i.e. if
+	 * the class itself or one of its superclasses carries such an annotation.
```

Comment:
Wondering if `carries` is the right word here. As stated here, I wouldn't be sure the annotations on methods and properties are considered. It might be just me so feel free to ignore :).